Classes/Controller/AbstractController.php

```diff
@@ -121,7 +121,7 @@ protected function initialize(RequestInterface $request): void
     {
         /** @var Request $request */
         $this->requestData = $request->getQueryParams()['tx_dlf'] ?? [];
-        $this->pageUid = (int) ($this->requestData['id'] ?? 0);
+        $this->pageUid = !($this instanceof OaiPmhController) ? $this->request->getAttribute('routing')->getPageId() : 0;
```

Comment:
This change breaks the TYPO3 backend which now throws a runtime exception as soon as "New Tenant" is called:
```
Error: Typed property TYPO3\CMS\Extbase\Mvc\Controller\ActionController::$request must not be accessed before initialization
```